src/Utilities/TypeTraits/CreateGetStaticMemberVariableOrDefault.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+/*!
+ * \ingroup TypeTraitsGroup
+ * \brief Generate a metafunction that will retrieve the specified
+ * `static constexpr` member variable, or, if not present, a default.
+ *
```

Comment:
I'm a bit worried that this helper makes it easier to create hidden defaults. How about adding a comment, something like this: `\warning Please use defaults responsibly. In many cases it is better to require the static member variable is present so the functionality isn't hidden. Also make sure to document the compile-time interface, e.g. with a \ref protocols "protocol".`

Reply:
I'll do this and add it to the type alias version as well.